lib/elixir/lib/process.ex

```diff
@@ -940,6 +940,25 @@ defmodule Process do
   @spec unalias(alias) :: boolean
   defdelegate unalias(alias), to: :erlang
 
+  @doc """
+  Add a descriptive term to the current process.
+
+  The term does not need to be unique, and in Erlang/OTP 27+ will be shown in Observer.
+
+  ## Examples
+
+      Process.set_label(:worker)
+      #=> :ok
```

Comment:
Perhaps a non-atom example as well to illustrate the point above?
```suggestion
      Process.set_label(:worker)
      #=> :ok
      
      Process.set_label({:any, "term"})
      #=> :ok
```